content/reference/templates/_index.md

```diff
@@ -126,7 +126,7 @@ Similarly, provided a channel `$channel`, `$channel.Name` gives the name of the
 |.IsMessageEdit| Returns boolean true/false if message is edited and edit trigger for custom commands is enabled. Defaults to false.|
 |.IsPremium| Returns boolean true/false whether guild is premium of YAGPDB or not.|
 |.LinkRegex| Returns string value of in-built link-matching regular expression.|
-|.Permissions| Returns all mapped-out permission bits available for Discord in their bitshifted decimal values; <br>e.g. `{{.Permissions.AddReactions}}` would return `64`, same as `{{bitwiseLeftShift 1 6}}`. More [here](https://discord.com/developers/docs/topics/permissions#permissions).|
+|.Permissions| Returns all mapped-out permission bits available for Discord in their bitshifted decimal values; <br>e.g. `{{.Permissions.AddReactions}}` would return `64`, same as `{{bitwiseLeftShift 1 6}}`. More in [Discord Permissions documentation](https://discord.com/developers/docs/topics/permissions#permissions).|
```

Comment:
```suggestion
|.Permissions| Returns all mapped-out permission bits available for Discord in their bitshifted decimal values; <br>e.g. `{{.Permissions.AddReactions}}` would return `64`, same as `{{bitwiseLeftShift 1 6}}`. More in [Discord's Permissions documentation](https://discord.com/developers/docs/topics/permissions#permissions).|
```
Whilst both approaches seem correct, I think this way it is clearer that this is

1. an external link
2. said external link goes to Discord's documentation
3. we have nothing to do with what's on that page